README.md

```diff
@@ -315,6 +315,17 @@ guarantee that they exist for future transactions.
 
 To delete a bucket, simply call the `Tx.DeleteBucket()` function.
 
+You can also iterate over all existing buckets with `Tx.forEach()`:
```

Comment:
Note `tx.ForEach` doesn't loop through nested buckets.
```suggestion
You can also iterate over all existing top-level buckets with `Tx.forEach()`:
```

It would be perfect if you can add an unit test `TestTx_ForEach_Toplevel_Buckets` in `tx_test.go`. It can be done in a separate PR if you are interested.